src/components/layout/footer.js

```diff
@@ -264,7 +264,9 @@ const SocialWrapperComponent = ({ is_career_page }) => {
             image_alt: `instagram ${alt_string}`,
         },
         {
-            link: 'https://www.linkedin.com/company/derivdotcom/',
+            link: is_career_page
+                ? 'https://www.linkedin.com/company/derivdotcom/life/'
+                : 'https://www.linkedin.com/company/derivdotcom/',
```

Comment:
lets put all the links in url constants?

Reply:
Done @mustofa-binary